okio/src/appleMain/kotlin/okio/ByteString.kt

```diff
@@ -22,10 +22,13 @@ import platform.posix.memcpy
 
 fun NSData.toByteString(): ByteString {
   val data = this
+  val size = data.length.toInt()
```

Comment:
if `ByteString.EMPTY` is available, let's return that directly when the size is 0.

Reply:
It is. Thanks